react/Shelf.js

```diff
@@ -132,7 +185,7 @@ Shelf.schema = {
 Shelf.defaultProps = {
   maxItems: 10,
   scroll: 'BY_PAGE',
-  arrows: true
+  arrows: true,
```

Comment:
this comma is right?

Reply:
@brunojdo the eslint warns about this comma